tibber_aws/aws_logs.py

```diff
@@ -0,0 +1,152 @@
+import logging
+from dataclasses import dataclass
+from datetime import datetime as dt
+import json
+from .aws_base import AwsBase
+from typing import Dict, List
+from types_aiobotocore_logs.client import CloudWatchLogsClient
+
+logger = logging.getLogger("aws_logs")
+
+
+@dataclass
+class CloudWatchLogEvent:
+    """An AWS CloudWatch log event (a log line)"""
+    logStreamName: str
+    timestamp: int
+    message: str
+    ingestionTime: int
+    eventId: str
+
+    def __post_init__(self):
+        self.timestamp_dt = dt.fromtimestamp(self.timestamp / 1000)
+        self.ingestionTime_dt = dt.fromtimestamp(self.ingestionTime / 1000)
+        self.message = json.loads(self.message)
+
+
+class Logs(AwsBase):
+    """CloudWatch Logs client to query AWS logs from a log stream."""
+    _client: CloudWatchLogsClient
+
+    def __init__(self, region_name="eu-west-1") -> None:
+        super().__init__("logs", region_name)
+
+    async def close(self):
+        await self._client.close()
```

Comment:
```suggestion
        await self._client.close()
        await super().close()
```
Maybe?

Reply:
actually the `AwsBase` close is enough! I'll just remove it 😄